test/FixedDataTableRoot-test.js

```diff
@@ -345,8 +345,8 @@ describe('FixedDataTableRoot', function () {
           let scrollableCellGroup = cellGroupElements[1]; // The scrollable columns group (the middle index of [fixed left, scrollable, fixed right])
 
           assert.equal(
-            scrollableCellGroup.style.right,
-            `${scroll}px`,
+            scrollableCellGroup.style.transform,
+            `translate(${-scroll}px,0px)`,
```

Comment:
This version of jsdom support CSSTransforms, hence instead of `left` and `right` we are using transform property.